### PR TITLE
Add deployment via Docker (fixes #10)

### LINKED ISSUE #10: Deployment via Docker
When reading the prerequisites, this application requires to set up and understand a lot of stuff. Would it be possible to provide a Docker image? That way, this could be run easily with just one command (`docker-compose up -d`) and a docker-compose.yml:

```yaml
version: "3"
services:
  app:
    image: manuel-ubert/boodle
    restart: unless-stopped
    volumes: ["./config.edn:/data/conf/config.edn"]
    ports: ["8080:8080"]
  db:
    image: postgres
    restart: unless-stopped
    environment:
      POSTGRES_PASSWORD: changeme
      POSTGRES_DB: boodle
```

I can create a pull request with a Dockerfile if you'd like to add Docker support, then you only need to add an automated build on https://cloud.docker.com.

### PR DESCRIPTION
**Pull request by moqmar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,12 @@ for more details.
 
 ## Usage
 
+#### The fast way: using Docker Compose
```
Comment (manuel-uberti) [inline README.md]: Could you use `###` instead of `####` here?

**Comment by manuel-uberti** [comment]
Thanks again for taking the time to work on this. Whenever you feel it's ready, let me know.

File: README.md (hunk 1 above)
Comment (moqmar) [reply README.md]: Whoops, you're right. :)

**Comment by moqmar** [comment]
Alright, just tested this on my Up Board, and it seems to work fine.

**Comment by manuel-uberti** [comment]
Great stuff. Can I ask you to add a link to docker/docker-compose website? Just as a reference in case someone don't know what we are talking about. :)

**Comment by manuel-uberti** [comment]
Also, can you squash the commits to a single one? Thank you.

**Comment by moqmar** [comment]
Alright, all done :)

**Comment by manuel-uberti** [comment]
Thanks. Going to merge and see if I can sort it out on Docker Hub.

**Comment by manuel-uberti** [comment]
@moqmar I am running the build here: https://cloud.docker.com/repository/docker/manueluberti/boodle

Is everything as you expected? Do we need other changes?

**Pull request merged**